File: arrays/richestCustomerWealth.java
package arrays;

public class richestCustomerWealth {
    public static void main(String[] args) {
        SolutionRCW solution = new SolutionRCW();
        System.out.println(solution.maximumWealth(new int[][] {{1,5}, {7,3 },{3,5}}));
        
    }   
}

class SolutionRCW {
    public int maximumWealth(int[][] accounts) {
        int max = 0,sum =0;
        for (int people = 0 ; people <= accounts.length -1;people ++){
            sum =0;
            for (int account =0; account <= accounts[people].length-1; account++){
                sum += accounts[people][account];   
            }
            if (sum>max) max = sum;
        }
        return max;
    }
}

File: arrays/richestCustomerWealth.py
#https://leetcode.com/problems/richest-customer-wealth/

def maximumWealth(accounts):

    #one liner
    # return max(sum(i) for i in accounts)

    maxx=0
    for i in accounts:
        #both below are equivalent
        
        # maxx= max(maxx, sum(i))
        maxx= maxx if (maxx> sum(i)) else sum(i)
    return maxx

#input
accounts = [[1,2,3],[3,2,1]]
print(maximumWealth(accounts))